torch/stable/library.h

```diff
@@ -0,0 +1,149 @@
+// this file can only have stable stuff! Akin to shim.h
+// but unlike shim.h, this file can contain header-only C++
+// code for better UX.
+
+#include <torch/csrc/inductor/aoti_torch/c/shim.h>
+
+
+template <typename T>
+StableIValue from(T val) {
+  static_assert(
+      sizeof(T) <= sizeof(StableIValue),
+      "StableLibrary stack does not support parameter types larger than 64 bits.");
+  return *reinterpret_cast<StableIValue*>(&val);
+}
+
+template <typename T>
+T to(StableIValue val) {
+  return *reinterpret_cast<T*>(&val);
+}
+// end to helpers for converting between StableIValue and actual IValues
+
+
+class TORCH_API StableLibrary final {
+  private:
+    TorchLibraryHandle lib_;
+  public:
+    enum Kind {
```

Comment:
No C-style enums in 2025 please
```suggestion
    enum class Kind {
```